TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/managers/BaseInstanceManager.java

```diff
@@ -305,19 +309,63 @@ private void startServer(BaseServerObject server) {
                 throw new ServerStartException("Could not start process");
             }
 
-            TimoCloudBase.getInstance().getSocketMessageManager().sendMessage(Message.create()
-                    .setType(MessageType.BASE_SERVER_STARTED)
-                    .setTarget(server.getId())
-                    .set("port", serverPort)
-                    .set("publicKey", RSAKeyUtil.publicKeyToBase64(publicKey))
-            );
-
+            //delay because we can only get the pid after 20 Millis.
+            new java.util.Timer().schedule(
```

Comment:
Please use the lambda syntax: `schedule(() -> {...})`

Reply:
This is not supported by java 8 as far as I know

Reply:
Ok it generally works of course, but not with TimerTask: https://stackoverflow.com/a/37970821/6494319

Reply:
Should I replace the TaskTimer then?